telegram/_chat.py

```diff
@@ -355,6 +359,92 @@ def de_json(cls, data: Optional[JSONDict], bot: "Bot") -> Optional["Chat"]:
 
         return super()._de_json(data=data, bot=bot, api_kwargs=api_kwargs)
 
+    def mention_markdown(self, name: str = None) -> str:
+        """
+        .. versionadded:: 20.0
+
+        Note:
+            :tg-const:`telegram.constants.ParseMode.MARKDOWN` is a legacy mode, retained by
+            Telegram for backward compatibility. You should use :meth:`mention_markdown_v2`
+            instead.
+
+        Args:
+            name (:obj:`str`): The name used as a link for the chat. Defaults to :attr:`full_name`.
+
+        Returns:
+            :obj:`str`: The inline mention for the chat as markdown (version 1).
+
+        """
+        full_name = self.full_name
+        if self.type == self.PRIVATE:
+            if name:
+                return helpers_mention_markdown(self.id, name)
+            if full_name:
+                return helpers_mention_markdown(self.id, full_name)
+            raise TypeError("Can not create a link to a private chat without first name")
```

Comment:
Imo we end our errors with a .

Oh and I would call it `create a mention`, not link, since link is already the attribute.